src/gsy_e/models/strategy/predefined_load.py

```diff
@@ -110,7 +110,7 @@ def event_market_cycle(self):
         self._read_or_rotate_profiles()
         super().event_market_cycle()
 
-    def _update_energy_requirement_future_markets(self):
+    def _update_energy_requirement_spot_market(self):
```

Comment:
NIT: Now that we have two different methods, I would propose to add a wrapper methods that contains both in order not to call `_update_energy_requirement_future_markets` inside of `_update_energy_requirement_spot_market`
```python
def _update_energy_requirement_in_state(self):
    self._update_energy_requirement_spot_market()
    self._update_energy_requirement_future_markets()
```